express-post-json/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+const grades = {};
+const gradesArray = [];
+
+app.get('/api/grades', (req, res) => {
+  res.json(gradesArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  gradesArray.push(grades[nextId]);
+  res.status(201).json(grades);
+  nextId++;
```

Comment:
For the POST route, we want to send back the added grade object and not the whole data model